src/config.c

```diff
@@ -636,6 +636,19 @@ CONFIG_GETTER(getBGIndexSleepGap) {
 CONFIG_BOOLEAN_SETTER(set_PrioritizeIntersectUnionChildren, prioritizeIntersectUnionChildren)
 CONFIG_BOOLEAN_GETTER(get_PrioritizeIntersectUnionChildren, prioritizeIntersectUnionChildren, 0)
 
+CONFIG_SETTER(setIndexCursorLimit) {
+  long long limit;
+  int acrc = AC_GetLongLong(ac, &limit, AC_F_GE0);
```

Comment:
@kei-nan If not using `AC_F_GE1` does zero have any special meaning? Currently it seems not.
We don't intend to make it unlimited, right?

Reply:
No intention to make it unlimited, 0 currently means we won'tt allow cursors to be opened.